src/components/MenuItem.tsx

```diff
@@ -402,6 +408,8 @@ function MenuItem(
         iconRight = Expensicons.ArrowRight,
         furtherDetailsIcon,
         furtherDetails,
+        furtherDetailsNumberOfLines,
```

Comment:
Should we set the default value to 2 since it was used earlier?

Reply:
You're right. Updated